### PR TITLE
Fix typo and optimize UT

### PR DESCRIPTION
**Pull request by Oliverwqcwrw**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

File: apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java [hunk 1]
```diff
@@ -47,7 +47,7 @@
     private final Counter sum;
 
     /**
-     * Max amount in this summary
+     * MaxAmount in this summary
```
Comment (wu-sheng) [inline apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java:50]: What does this mean?

Comment (Oliverwqcwrw) [reply apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java:50]: I think it's a little clearer

Comment (wu-sheng) [reply apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java:50]: But your changes are using two different styles.

Comment (Oliverwqcwrw) [reply apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java:50]: Sorry,I didn't catch your meaning

Comment (Oliverwqcwrw) [reply apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java:50]: You can change it to whatever you feel is more appropriate

Comment (wu-sheng) [reply apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java:50]: Personally, I prefer don't change this one.

**Comment by wu-sheng** [comment]
I am holding this PR to make more of your chore fixes to be merged in one pull request.

File: apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java (hunk 1 above)
Comment (Oliverwqcwrw) [reply apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java:50]: All right, let me undo it

**Comment by Oliverwqcwrw** [comment]
> I am holding this PR to make more of your chore fixes to be merged in one pull request.

OK

**Comment by Oliverwqcwrw** [comment]
I have nothing to modify, please check it

File: apm-sniffer/apm-sdk-plugin/dbcp-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/dbcp/v2/define/DelegatingConnectionInstrumentation.java [hunk 2]
```diff
@@ -29,8 +29,7 @@
 import static org.apache.skywalking.apm.agent.core.plugin.match.NameMatch.byName;
 
 /**
- * DBCP use DelegatingConnection which is the delegating implementation of Connection.All of
- * the methods from the Connection interface simply check to see that the Connection is active.
+ * DBCP use DelegatingConnection which is the delegating implementation of Connection.All the methods from the Connection interface simply check to see that the Connection is active.
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/dbcp-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/dbcp/v2/define/DelegatingConnectionInstrumentation.java:32]: Don't reformat the comment.

Comment (Oliverwqcwrw) [reply apm-sniffer/apm-sdk-plugin/dbcp-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/dbcp/v2/define/DelegatingConnectionInstrumentation.java:32]: ok,I will change it

**Comment by wu-sheng** [comment]
image

You changed this. I think you did it locally, but you didn't know

**Comment by wu-sheng** [comment]
When you merge with the upstream, and submodule changed, your git may say something changed, you should reset it rather than submitting the change.

**Comment by Oliverwqcwrw** [comment]
Can this PR be merged now?

**Comment by wu-sheng** [comment]
You need to rollback the submodule. I think this is very basic skill of git, you should try to learn it.

**Comment by Oliverwqcwrw** [comment]
Ok

**Comment by Oliverwqcwrw** [comment]
My localhost exec `git -c protocol.version=2 submodule update --init --force --depth=1` is no problem

> Submodule path 'apm-protocol/apm-network/src/main/proto': checked out '8095caa8a003cd2acd3a8a5f1d0e1739dc177ce6'

But why there appears the problem

**Comment by wu-sheng** [comment]
You could copy required files into your another PR if you have to.
I am not using git command, most using SourceCodes, so, don't know what is your local issue

**Comment by Oliverwqcwrw** [comment]
Ok,I will move these modified files to another PR

**Comment by Oliverwqcwrw** [comment]
I have moved these changes to #148 ,so I will close this PR

**Pull request closed without merging**